pkgdown/_pkgdown.yml

```diff
@@ -12,8 +12,6 @@ authors:
     href: http://hadley.nz
   Posit Software, PBC:
     href: https://www.posit.co
-    html: >-
```

Comment:
I asked our graphic designer, and she suggested just using text since we're not really supposed to use the logo like this anyway.